mipengine/node/monetdb_interface/guard.py

```diff
@@ -111,6 +111,8 @@ class InvalidSQLParameter(Exception):
 datamodel_ptrn = re.compile(datamodel_re, re.A)
 sqlidentifier_re = r"[a-z_][a-z0-9_]*"
 datatable_ptrn = re.compile(rf'"{datamodel_re}"\.("\w+"|{sqlidentifier_re})', re.A)
+uuid_re = r"[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}"
+uuid_ptrn = re.compile(uuid_re, re.A)
```

Comment:
Actually it just occurred to me that uuids are composed of hex characters. This means that both lowercase and uppercase letters are valid, e.g. `123AF` is valid hex. 
If we want to be sure that all uuids will match there are two solutions:
1. The char set in the regex becomes `[0-9a-fA-F]`
2. `re.compile` takes one additional flag `re.I` (see [here](https://docs.python.org/3/library/re.html#re.I)). Note that to combine flags you have to pass `re.A | re.I`.